demo/build.gradle.kts

```diff
@@ -65,6 +65,6 @@ dependencies {
   implementation(Dependencies.timber)
   implementation(Dependencies.appAuth)
   implementation(Dependencies.jwtDecode)
-  implementation("com.google.android.fhir:engine:0.1.0-beta01")
+  implementation("com.google.android.fhir:engine:0.1.0-beta02")
   implementation("com.google.android.fhir:data-capture:0.1.0-beta03")
```

Comment:
could you also update the data-capture lib to 1.0.0?

Reply:
Was going to do in a separate PR once I understand how the new QuestionnaireFragment builder pattern works, or if someone else can help. The engine changes were very quick in comparison.

Reply:
Sure, Ill approve this then, and the follow up for the QuestionnaireFragment builder. Will help out if you need it